runtime/syntax/confini.vim

```diff
@@ -8,5 +8,8 @@ endif
 
 " Use the cfg syntax for now, it's similar.
 runtime! syntax/cfg.vim
+" Only accept '#' as the start of a comment.
+syn clear CfgComment
+syn match CfgComment "#.*"
```

Comment:
This can probably also have `contains=@Spell`?

Reply:
makes sense. I make it like this then.